apps/condo/domains/common/utils/serverSchema/changeTrackable.js

```diff
@@ -305,7 +309,7 @@ const buildDataToStoreChangeFrom = async (args) => {
                     existingItem,
                     originalInput,
                 })
-                if (difference(existing.ids, updated.ids).length > 0) {
+                if (difference(existing.ids, updated.ids).length > 0 || difference(updated.ids, existing.ids).length > 0) {
```

Comment:
Maybe u can use xor here?

For commutative symmetric difference (unique values in either array), use _.xor([arrays]) instead.